overlord/snapstate/snapmgr.go

```diff
@@ -139,6 +139,11 @@ type SnapState struct {
 	// InstanceKey is set by the user during installation and differs for
 	// each instance of given snap
 	InstanceKey string `json:"instance-key,omitempty"`
+
+	// RefreshInhibitedime records the time when the refresh was first
+	// attempted but inhibited because the snap was busy. This value is
+	// reset on each successful refresh.
+	RefreshInhibitedTime time.Time `json:"refresh-inhibited-time,omitempty"`
```

Comment:
you don't want this; it'll dirty up the state with useless `"refresh-inhibited-time":"0001-01-01T00:00:00Z"` (`time.Time{}` is never "empty" in the json sense). See https://play.golang.org/p/jeQaXjjvFqI.

You want it to be a `*time.Time`, instead.

Reply:
Done now.